src/screens/OtherProfile/index.js

```diff
@@ -175,7 +175,8 @@ const OtherProfile = () => {
   const handleSetUnFollow = async () => {
     setDataMain((prevState) => ({
       ...prevState,
-      is_following: false
+      is_following: false,
+      is_me_following_target: false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `is_me_following_target` state variable seems to be a good approach for tracking if the current user is following the target user. However, it's important to ensure that this state variable is updated in all places where the follow/unfollow actions can occur.

```diff
     setDataMain((prevState) => ({
       ...prevState,
       is_following: false,
+      is_me_following_target: false
     }));
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->